crates/re_viewer/src/app.rs

```diff
@@ -1366,6 +1366,37 @@ fn memory_use_label_ui(ui: &mut egui::Ui, gpu_resource_stats: &WgpuResourcePoolS
             format_number(gpu_resource_stats.num_textures),
             format_number(gpu_resource_stats.num_buffers),
         ));
+    } else {
```

Comment:
We should also consider adding a fallback on the RSS, as found by `bytes_resident()` in `crates/re_memory/src/memory_use.rs`. Then we would never need to suggest the `AccountingAllocator`

Reply:
Btw, https://github.com/Arc-blroth/memory-stats/issues/2 is fixed, so we should be able to get the RSS on Linux too (please check!)

Reply:
We know have fallbacks for all cases; but I still recommend setting up an AccountingAllocator because RSS is, as always, completely off the mark